test/models/report_test.rb

```diff
@@ -3,7 +3,51 @@
 require 'test_helper'
 
 class ReportTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  test 'editable?' do
+    alice_report = reports(:alice)
+    alice = users(:alice)
+    bob = users(:bob)
+
+    assert alice_report.editable?(alice)
+    assert_not alice_report.editable?(bob)
+  end
+
+  test 'created_on' do
+    alice_report = reports(:alice)
+
```

Comment:
```suggestion
    alice_report.created_at = ...
```

というようにこのタイミングでcreated_atに値を設定するのがいいかも。
（通常は静的な値は入れないが、このテストだけ例外的にOKとする）

Reply:
fixturesで作成した日報の``created_at``の値が現在時刻を持つということを理解したので、静的な値を入れないで下記の様に修正しました。
```
assert_equal Date.current, alice_report.created_on
```